Automattic/wp-calypso
# Change contact info block icon to account box

Due to an issue in Calypso core, https://github.com/WordPress/gutenberg/issues/13010, there is no space after a parent block name when choosing sub blocks

<img width="123" alt="screen shot 2019-03-04 at 11 00 43 am" src="https://user-images.githubusercontent.com/3246867/53747789-0bff8600-3e72-11e9-9d51-80a4da8ec7bf.png">

This PR changes the icon to use the account_box so that there is a space after the parent block name

<img width="516" alt="screen shot 2019-03-04 at 11 34 18 am" src="https://user-images.githubusercontent.com/3246867/53747844-289bbe00-3e72-11e9-9246-144084963aa6.png">

#### Testing instructions

Add a Contact Info block to a post. Click within it to add another block, and you should see the second image/icon above with space between the icon and block name instead of the first.
